simulariumio/tests/converters/test_cytosim_converter.py

```diff
@@ -115,13 +116,13 @@ def test_box_size_provided(box_size, expected_box_size):
     assert box_size == expected_box_size
 
 
+auto_scale_factor = VIEWER_DIMENSION_RANGE.MIN / (0.400052 + 0.001)
```

Comment:
where do these numbers come from?

Reply:
they're pulled from the test data, that represents the max range of the positional data, which determines the scale factor. I can pull that out and call it the range, so it's a little more clear?

- [ ] make range variable